app/templates/account/donate.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+<div class="ui stackable centered grid container">
```

Comment:
indent everything one more so that this first `<div>` is one indent further than the `{% block content %}`